apps/nextjs-website/react-components/components/Editorial/Editorial.tsx

```diff
@@ -0,0 +1,77 @@
+import Grid from '@mui/material/Grid';
+import Stack from '@mui/material/Stack';
+import ContainerRC from '../common/ContainerRC';
+import { EditorialProps } from '../../types/Editorial/Editorial.types'
+import { useIsMobile } from './Editorial.helpers';
+import { BackgroundColor } from '../common/Common.helpers';
+import { Content as EditorialContent } from './Content';
+import { Ctas as EditorialCtas } from './Ctas';
+import { Image as EditorialImage } from './Image';
+
+const Editorial = (props: EditorialProps) => {
+  const {
+    image,
+    eyelet = '',
```

Comment:
I don't think eyelet should have a default value.

The eyelet component in `Editorial.helpers.tsx` relies on it being undefined or null to return null.

If we were to pass it an empty string like the above it would create an empty eyelet, which could cause issues with margins and such.